src/components/button/style.css

```diff
@@ -37,6 +37,9 @@ button {
 	padding: 0.8em;
 	min-width: 8em;
 }
+:host(:not([size=icon])):has(smoothly-icon) > button {
+	padding: 0.6em;
+}
```

Comment:
This is needed to make the button equally high if there is an icon in it or text

Reply:
is this only true for icon size=small?

Reply:
its true everytime is not icon